examples/README.md

```diff
@@ -0,0 +1,7 @@
+# aimlflow Examples
+
+This folder contains a pre-made example script to quickly get started with.
+
+## Example
+
+1. Hyperparameter tuning - This example uses `pytorch` and `Ray Tune` to tune 3 hyperparameters, with 4 different batch sizes. `MLflow` is used as the parameter tracking tool. Later on we convert runs into `Aim` format to give the possibility of exploring the runs on `Aim UI`.
```

Comment:
> "Later on we convert runs..."

Lets re-phrase this a bit: "In order to start exploring the runs on `Aim UI` we convert runs into `Aim` storage and watch for new mlflow logs."